Dockerfile

```diff
@@ -1,3 +1,20 @@
+FROM gradle:7-jdk11-alpine AS build
+
+COPY --chown=gradle:gradle . /build
+WORKDIR /build
+
+ARG VERSION_TAG
+RUN ./gradlew shadowJar \
+    --no-daemon \
+    -Prelease.forceVersion=$VERSION_TAG
+
+
 FROM openjdk:11
-COPY main/build/libs/*.jar /
+COPY --from=build /build/main/build/libs/*.jar /
 WORKDIR /
+
+ARG VERSION_TAG
+ENV VERSION_TAG=$VERSION_TAG
+RUN echo "#!/bin/bash\nexec java -jar /gtfs-validator-${VERSION_TAG}-SNAPSHOT-cli.jar \"\$@\"" > /entrypoint.sh \
```

Comment:
I'm not sure there will always be a -SNAPSHOT suffix for the jar, depending on the version number passed in.  Again, another reason to avoid encoding the version number in the jar here.

Reply:
I did some digging on this, and it seems like the underlying lib offers a `forceSnapshot` option to force the `-SNAPSHOT` suffix, but none to disable it. It seems like as long as we're not feeding it a git repo + clean working tree + a tag that matches HEAD, it's always going to append this